### ISSUE TITLE
False positive with `require-computed-macros` rule using `this.` for filterBy/mapBy value

### ISSUE THREAD

**Issue by bmish** [ember-cli/eslint-plugin-ember#869]
This example should not use the `filterBy` macro because `this.token` needs to remain inside a function:

```js
myProp: computed('items.@each.token', 'token', function () {
  return this.items.filterBy('token', this.token);
}),
```
